docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+---
+version: '3'
+services:
+  clickhouse:
+    image: yandex/clickhouse-server
```

Comment:
can we use pure `docker run` from within travis, instead of having docker-compose?

Reply:
Yes, but why, docker-compose is a very simple tool for running containers?

Reply:
I prefer not to use Docker Compose, since we're running single command anyway.

With the whole container community movement, we might as well use some CRIO based runtime in the future, not Docker itself.